packages/fluent-ui/package.json

```diff
@@ -28,7 +28,7 @@
     "node": ">=14"
   },
   "peerDependencies": {
-    "@fluentui/react": "^7.190.3",
+    "@fluentui/react": "^8.106.9",
```

Comment:
Is this the earliest version of `fluent` v8 that will actually work? Locking peerDependencies down may not be the best thing. Also this may be considered a breaking change if the 7.x version of Fluent won't work with these changes. Would making this `>=7` cause issues?

Reply:
I believe it can be set to `^8.1.0`. It there is a definitive way for me to test this though?

Also, the changes here are to address breaking changes in `v8`. How are breaking changes handled for theme implementations?

Reply:
With my latest commit, this became `>= 7`.